content/en/docs/_index.md

```diff
@@ -29,5 +29,68 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
+
+If a dependency is ever discovered, OLM will attempt to resolve said dependency by searching through known `CatalogSources` for an operator that `owns` the API. If an operator is found that `owns`the `required` API, OLM will attempt to deploy the operator that `owns` the API. If no `owner` is found, OLM will fail to deploy the operator with the dependency.
+
+### Work similar to package Managers
+
+OLM verifies that all require APIs are available for the operator. If it is not available, then it will not install the operator.
+
+OLM will never update an operator in a way that breaks another that depends upon it.
+
+### Operator Groups
+
+OLM introduces the concept of `OperatorGroups` to enable cluster admins complete control over the permissions that OLM grants operators that it deploys.
+
+With `OperatorGroups`, a cluster admin can:
+
+* Define the set of permissions that OLM may grant to member operators
+* Define the set of namespaces that OLM may grant namespaced permissions in.
+
+## Building blocks of OLM
+
+### [Install OLM](https://github.com/laxmikantbpandhare/olm-docs/blob/master/content/en/docs/getting-started/_index.md)
+
+OLM requires you to provide metadata about your operator in order to ensure that it can be kept running safely on a cluster, and to provide information about how updates should be applied as you publish new versions of your operator. This is very similar to packaging software for a traditional operating system.
+
+### [Package Validation](https://github.com/laxmikantbpandhare/olm-docs/blob/master/content/en/docs/Tasks/validate-package.md)
+
+Once you've [created your operator's package manifests](https://github.com/laxmikantbpandhare/olm-docs/blob/master/content/en/docs/Tasks/creating-operator-manifests.md), you will want to ensure that your package is valid and in the correct format. To ensure this, you should take several steps to ensure that your package can be used to install your operator in OLM.
+
+### [Publish Package into catalog](https://github.com/laxmikantbpandhare/olm-docs/blob/master/content/en/docs/Tasks/make-operator-part-of-catalog.md)
+
+We will publish the package into a catalog, install that catalog onto a Kube cluster, and then install the operator onto that cluster. If all of that succeeds and your operator is behaving as expected, your package is valid.
+
+### [Subscriptions](https://github.com/laxmikantbpandhare/olm-docs/blob/master/content/en/docs/Concepts/crds/subscription.md)
+
+A Subscription represents an intention to install an operator. Subscriptions are Custom Resources that relate an operator to a CatalogSource. Subscriptions describe which [channel](/docs/glossary/#channel) of an operator package to subscribe to and whether to perform updates automatically or manually. If set to automatic, the Subscription ensures OLM will manage and upgrade the operator to ensure the latest version is always running in the cluster.
```

Comment:
THIS is exactly what I'd expect in a "Building blocks of OLM", basically concepts that exist in OLM an what they do.

Reply:
Yup, I will add more concepts that exist in OLM and their main purpose.